skimage/draw/_draw.pyx

```diff
@@ -347,18 +342,14 @@ def circle_perimeter(Py_ssize_t cy, Py_ssize_t cx, Py_ssize_t radius,
            [0, 0, 0, 1, 1, 1, 0, 0, 0, 0],
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
-
     """
 
     cdef list rr = list()
     cdef list cc = list()
 
     cdef Py_ssize_t x = 0
-    cdef Py_ssize_t y = radius
-    cdef Py_ssize_t d = 0
-
-    cdef double dceil = 0
-    cdef double dceil_prev = 0
+    cdef Py_ssize_t y = <int> round(radius)
```

Comment:
@stefanv @sciunto I tried to search about these methods for non integer radius and center but couldn't found much. Can you tell me which approximation method should I apply here?